h2o-core/src/main/java/hex/Model.java

```diff
@@ -2609,20 +2609,37 @@ protected SBPrintStream toJavaTransform(SBPrintStream ccsb,
   // is well, false is there are any mismatches.  Throws if there is any error
   // (typically an AssertionError or unable to compile the POJO).
   public boolean testJavaScoring(Frame data, Frame model_predictions, double rel_epsilon) {
-    return testJavaScoring(data, model_predictions, rel_epsilon, 1e-15, 0.1);
```

Comment:
I am wandering if `rel_epsilon` in `JavaScoringOptions ` would have any benefit. At least it would be consistent with all the other options if  `rel_epsilon ` would be in `JavaScoringOptions`.

Reply:
rel_epsilon is the only mandatory parameter, the other can be optional

I think you don't want the test to be calling testJavaScoring without specifying/showing in code the precision it is working with - it is good when you see it explicitly